XiEditor/AppDelegate.swift

```diff
@@ -171,6 +171,8 @@ class AppDelegate: NSObject, NSApplicationDelegate, XiClient {
                                                   textColor: theme.foreground)
         }
     }
+    
+    var languageId: String?
```

Comment:
In general when I'm writing Cocoa stuff I adhere to the Cocoa naming conventions, which prefer spelling out whole names, so 'Identifier' instead of 'Id'. We aren't super consistent about this, so 🤷‍♂️, just a note.